service/history/timerQueueProcessorBase.go

```diff
@@ -214,17 +214,12 @@ func (t *timerQueueProcessorBase) notifyNewTimers(
 		return
 	}
 
-	isActive := t.scope == metrics.TimerActiveQueueProcessorScope
-
 	newTime := timerTasks[0].GetVisibilityTime()
 	for _, task := range timerTasks {
 		ts := task.GetVisibilityTime()
 		if ts.Before(newTime) {
 			newTime = ts
 		}
-
-		scopeIdx := getTimerTaskMetricScope(task.GetType(), isActive)
-		t.metricsClient.IncCounter(scopeIdx, metrics.NewTimerCounter)
```

Comment:
do we emit this counter somewhere else? or do we just drop it?

Reply:
transfer / replication / visibility side does not have similar metrics
plus we do not use this metrics